Source/Extensions/Blazorise.DataGrid/DataGridColumn.cs

```diff
@@ -17,7 +17,7 @@ public partial class DataGridColumn<TItem> : BaseDataGridColumn<TItem>
         private readonly Lazy<Func<TItem, object>> valueGetter;
         private readonly Lazy<Action<TItem, object>> valueSetter;
 
-        private Dictionary<DataGridSortMode, SortDirection> currentDirection { get; set; } = new Dictionary<DataGridSortMode, SortDirection>();
+        private Dictionary<DataGridSortMode, SortDirection> currentSortDirection { get; set; } = new Dictionary<DataGridSortMode, SortDirection>();

```

Comment:
Does this need to be a Dictionary to hold two items? Or could it be a fixed size array with like two value tuples?
I'm pretty sure the memory footprint is less, but we could measure it if in doubt.

Reply:
I used a dictionary because it is easier to access the current sort mode, eg. `currentSortDirection[mode]`. With a list, I would need to get the item, check if it's null, and then get the value.

Reply:
Is it not actually the same thing? Since it's for internal use, you do not need to do null checks.
You'd have the same null check problem with the Dictionary, if you did not assume it already has the item you are looking for.
Was talking about something like: `(DataGridSortMode sortMode, SortDirection sortDirection)[2]`.
(Don't know if syntax for array of ValueTuple is correcct)

Reply:
Let's just leave it as it is 😅